webapp/channels/src/tests/helpers/markdown_hotkey_helpers.ts

```diff
@@ -82,8 +95,9 @@ function makeLinkHotKeyEvent(input, start, end) {
  * @param  {function} getValue - single parameter for the React Component instance
  * NOTE: runs Jest tests
  */
-export function testComponentForMarkdownHotkeys(generateInstance, initRefs, find, getValue, intlInjected = true, targetId) {
-    const shallowRender = intlInjected ? shallowWithIntl : shallow;
+export function testComponentForMarkdownHotkeys(generateInstance: (input: string) => JSX.Element, initRefs: (instance: ShallowWrapper, setSelectionRange: (start: number, end: number) => void) => void, find: (instance: ShallowWrapper) => JSX.Element, getValue: (instance: React.Component<any, any>|ShallowWrapper) => string, intlInjected = true, targetId: string) {
```

Comment:
Just for legibility, I would separate this in several lines
```suggestion
export function testComponentForMarkdownHotkeys(
    generateInstance: (input: string) => JSX.Element,
    initRefs: (instance: ShallowWrapper, setSelectionRange: (start: number, end: number) => void) => void,
    find: (instance: ShallowWrapper) => JSX.Element,
    getValue: (instance: React.Component<any, any>|ShallowWrapper) => string,
    intlInjected = true,
    targetId: string,
) {
```